DUI3-DX/Connectors/ArcGIS/Speckle.Connectors.ArcGIS3/Bindings/ArcGISSendBinding.cs

```diff
@@ -112,17 +163,28 @@ public async Task Send(string modelCardId)
   /// <summary>
   /// Checks if any sender model cards contain any of the changed objects. If so, also updates the changed objects hashset for each model card - this last part is important for on send change detection.
   /// </summary>
-  private void RunExpirationChecks()
+  private void RunExpirationChecks(bool idsDeleted)
```

Comment:
This seems different from others, can you explain in the comments why it was needed? Maybe this is something we need for others too? It is important to know when we start to refactor this function to have unified functionality as in https://spockle.atlassian.net/browse/CNX-9576

Reply:
not sure if other connectors are affected in the same way, but basically this is needed to update the list of selected objectIDs saved in the model card. Otherwise, it can happen that all objects have been deleted but modelCard still have the list of Ids and prevents "No objects selected" warning on Send. Just sends an empty commit.